app/templates/partials/question.html

```diff
@@ -24,13 +24,55 @@ <h3 class="question__subtitle mars">{{question_subtitle}}</h3>
 {% endset %}
 
 {% set question_answers %}
-  <div class="question__answers">
-    {%- for answer in question.answers -%}
-      {% include theme('partials/answer.html') %}
-    {%- endfor -%}
+ <div class="question__answers">
+   {%- if question.schema_item.type == 'RepeatingAnswer' %}
+     {%- for answer in question.answers %}
+       <div class="question__answer js-household-person">
+         <h3 class="neptune">Person <span class="js-household-loopindex">{{ loop.index }}</span>
+           <small>(
+           {%- if loop.index == 1 -%}
+              <span class="js-household-action">{{ _('You') }}</span>
+           {%- else -%}
+           <button class="btn btn--link"
+              formaction="{{ url_for('action.remove_answer',
+                eq_id=meta.survey.eq_id,
+                collection_id=meta.survey.collection_id,
+                form_type=meta.survey.form_type,
+                location=content.id,
+                question=question.schema_item.id) }}"
+              name="action[remove_answer]" value="{{ loop.index0 }}">{{ _('Remove') }} <span class="u-vh">{{ _('Person') }} {{loop.index0}}</span></button>
+           {%- endif -%}
+           )</small>
+         </h3>
+         {% include theme('partials/answer.html') %}
+       </div>
+     {%- endfor %}
+   {%- else %}
+     {%- for answer in question.answers -%}
+      <div class="question__answer">
+       {% include theme('partials/answer.html') %}
+     </div>
+     {%- endfor -%}
+   {%- endif %}
+ </div>
+{% endset %}
+
+{% set question_actions %}
+  {%- if question.schema_item.type == 'RepeatingAnswer' %}
+  <div class="question__actions">
+    <button class="btn btn--border btn--secondary venus js-household-btn" type="submit"
+          formaction="{{ url_for('action.add_answer',
```

Comment:
Not sure how I feel able this, maybe we should just hard code the url with the dynamic parts substituted or create a add/remove variable in the context